app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -145,10 +145,7 @@ def stop_control_monitor
     end
 
     def verify_credentials(auth_type = nil, _options = {})
-      self.class.validate_connection do
-        with_provider_connection(:use_broker => false, :auth_type => auth_type) {}
-      end
-      true
+      self.class.raw_connect(:use_broker => false, :ems => self)
```

Comment:
I refactored this so we didn't have to implement the same validation code in two places